src/ArticleContent.jsx

```diff
@@ -9,19 +9,54 @@ export function ArticleContent() {
     const [articleContent, setArticleContent] = useState({});
     const [isLoading, setIsLoading] = useState(true);
     const [commentList, setCommentList] = useState([]);
+    const [articlePoints, setArticlePoints] = useState(null);
+    const [error, setError] = useState(null);
+
+    const handleDecrease = (event) => {
+        axios
+            .patch(`https://xqnews.onrender.com/api/articles/${article_id}`, {
+                inc_votes: -1,
+            })
+            .then((response) => {
+                setArticlePoints(response.data.article.votes);
+            })
+            .catch((error) => {
+                setError("There was an error decreasing the vote!");
+            });
+    };
+
+    const handleIncrease = (event) => {
+        axios
+            .patch(`https://xqnews.onrender.com/api/articles/${article_id}`, {
+                inc_votes: 1,
+            })
+            .then((response) => {
+                setArticlePoints(response.data.article.votes);
```

Comment:
Nice work. However, this isn't optimistically rendered yet. for this, you need to set the article points before the api call, then undo it if the catch block if reached. Otherwise very good 👍